peps/pep-0101.rst

```diff
@@ -388,8 +387,6 @@ and guides you to perform some manual steps.
     ``docs.nyc1.psf.io``. Make sure the files are in group ``docs`` and are
     group-writeable.
 
-  - Let the DE check if the docs are built and work all right.
```

Comment:
Arguably this step could be kept (modulo whoever does the checking), there have been recent releases where the docs *haven't* built properly, so it was important to check, even though CI failing/hanging alerted us to this. 

A

Reply:
Arguably at minimum, the HTML files should seem fine at dip-test level, the EPUB file should pass EPUBcheck, the PDF archives should contain the expected files & not be corrupt, etc. These feel like reasonable checks?

A

Reply:
+1, no need for **me** to do so, but if someone in the release team could just hit refresh on docs.python.org to see if everything is OK it would be nice.

There's multiple places in the PEP 101 that tells about "checking the doc" maybe it could be merged and pushed a bit downwards. like « If it's a new branch [...] verify that the [online docs](https://docs.python.org/) are building properly » no, no, verify, new branch or not, just check the page.

I'm glad we automated docsbuild-script to the point that almost all docs steps were removed from PEP 101.